dspace-oai/pom.xml

```diff
@@ -107,74 +92,56 @@
                     <groupId>org.apache.commons</groupId>
                     <artifactId>commons-lang3</artifactId>
 	        </exclusion>
-		<exclusion>
-		    <groupId>log4j</groupId>
-		    <artifactId>log4j</artifactId>
-	        </exclusion>
-		<exclusion>
-		    <groupId>org.slf4j</groupId>
-		    <artifactId>slf4j-log4j12</artifactId>
-		</exclusion>
+                <exclusion>
+                    <groupId>log4j</groupId>
+                    <artifactId>log4j</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.slf4j</groupId>
+                    <artifactId>slf4j-log4j12</artifactId>
+                </exclusion>
+                <!-- Later version provided by SolrJ -->
+                <exclusion>
+                    <groupId>org.codehaus.woodstox</groupId>
+                    <artifactId>wstx-asl</artifactId>
+                </exclusion>
             </exclusions>
         </dependency>
 
-        <!-- Java Injection and MVC -->
-        <dependency>
-            <groupId>javax.inject</groupId>
```

Comment:
This is still a direct dependency and should be retained.

Reply:
@mwoodiupui : good catch. At a glance, this looks to have been accidental (as I replaced several dependencies just after this one).  I'll restore it

Reply:
This has now been restored.  Marking comment as resolved